lib/manageiq/providers/ovirt/redhat_rhevm_probe.rb

```diff
@@ -0,0 +1,16 @@
+require 'manageiq/network_discovery/port_scanner'
+
+module ManageIQ::Providers::Ovirt
+  class RedHatRhevmProbe
+    def self.probe(ost)
+      require 'ovirt'
+      Ovirt.logger = $rhevm_log if $rhevm_log
+
+      if PortScanner.portOpen(ost, Ovirt::Service::DEFAULT_PORT)
+        if Ovirt::Service.ovirt?(:server => ost.ipaddr, :verify_ssl => false)
+          ost.hypervisor << :rhevm
```

Comment:
Does the `ost.hypervisor << :rhevm` belong in here?

I would expect to call `ManageIQ::Providers::Ovirt::Probe.probe(hostname or ip_address)` and have it return `true` or `false`.  Unless it could return many different things?

Reply:
Although I tend to agree with you, unfortunately the way probes are working so far is by passing `ost` structure which is updated by the probe themselves. Changing that is going to require a refactor across all probes, which is not the intend of this PR.